### PR TITLE
build/configs/stm: Enable FS_PROCFS & add dependency on tash commands

### PR DESCRIPTION
**Pull request by yashwanth686007**
The initial patch enables FS_PROCFS in stm configurations to eliminate build error. The second patch adds dependency of FS_PROCFS on tash commands "ps" and "heapinfo".

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [915fbfab9ce34e44419c309d5e2feb2dc12404db] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [915fbfab9ce34e44419c309d5e2feb2dc12404db] - Code Rule Check (C++) OK.

File: apps/system/utils/Kconfig [hunk 1]
```diff
@@ -172,7 +172,7 @@ config ENABLE_KILLALL_CMD
 config ENABLE_PS_CMD
 	bool "ps"
 	default y
-	depends on !FS_PROCFS_EXCLUDE_PROCESS
+	depends on !FS_PROCFS_EXCLUDE_PROCESS && FS_PROCFS
```
Comment (sunghan-chang) [inline apps/system/utils/Kconfig]: Let's change a sequence of them.
```
depends on FS_PROCFS && !FS_PROCFS_EXCLUDE_PROCESS
```

Comment (yashwanth686007) [reply apps/system/utils/Kconfig]: ok.Done.

**Comment by seinfra** [comment]
Target : [adfeeb901802b981fff186e401a54b86fad95e2d] - Code Rule Check (C++) OK.

**Comment by seinfra** [comment]
Target : [adfeeb901802b981fff186e401a54b86fad95e2d] - Code Rule Check OK.

**Pull request merged**